metriq_gym/poll_clops.py

```diff
@@ -0,0 +1,52 @@
+"""Dispatch a CLOPS job with CLI parameters to Qiskit and wait for result."""
+import json
+import logging
+import os
+
+from dataclasses import asdict
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+from metriq_gym.parse import parse_arguments
+from metriq_gym.process import poll_job_results
+from metriq_gym.bench import BenchJobResult, BenchJobType, BenchProvider
+from metriq_gym.hardware.clops_benchmark import clops_benchmark
+
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s")
+
+def main():
```

Comment:
Similar `def main` question here as the one put in `poll_qv.py`.

Reply:
Same answer.